src/anndata/io.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import sys
+from typing import TYPE_CHECKING
+
+from ._core.sparse_dataset import sparse_dataset
+from ._io.h5ad import read_h5ad, write_h5ad
+from ._io.read import (
+    read_csv,
+    read_excel,
+    read_hdf,
+    read_loom,
+    read_mtx,
+    read_text,
+    read_umi_tools,
+)
+from ._io.specs import read_elem, write_elem
+from ._io.write import write_csvs, write_loom
+
+if "zarr" in sys.modules or TYPE_CHECKING:
+    from ._io.zarr import read_zarr, write_zarr
+else:
+    # Importing zarr is slow, so unless it’s already imported,
+    # we wrap these functions into shims.
```

Comment:
Don't we import zarr at import time to define the `ZarrArray`/ `ZarrGroup` classes?

Reply:
Doesn’t look like it.

`PYTHONPROFILEIMPORTTIME=1 python -c 'import anndata' | complete | get stderr | pipx run tuna /dev/stdin`
![grafik](https://github.com/user-attachments/assets/d3b4cd3a-d139-4ad1-bc6a-82982b646683)

`PYTHONPROFILEIMPORTTIME=1 python -c 'import zarr; import anndata' | complete | get stderr | pipx run tuna /dev/stdin`
![grafik](https://github.com/user-attachments/assets/e6117dd6-b2aa-46ec-9046-cac476d7c4db)

Reply:
Maybe that was some weird version resolution, I noticed `uv` acting up in recent times.

Maybe I just got some old zarr version that took one second to import :thinking:

Reply:
So just need to update the comment?

Reply:
```suggestion
    # In case zarr is not imported (and maybe not installed),
    # wrap these functions into shims.
```